### PR TITLE
Image sent via assets v3 API will report as photo (WEBAPP-3335)

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/conversation/ConversationRepository.coffee [hunk 1]
```diff
@@ -2341,7 +2341,9 @@ class z.conversation.ConversationRepository
       is_ephemeral = false
 
     action_type = switch message_content_type
-      when 'asset' then 'file' if message.asset.original?
+      when 'asset'
+        if message.asset.original?
+         if message.asset.original.image? then 'photo' else 'file'
```
Comment (lipis) [inline app/script/conversation/ConversationRepository.coffee]:
``` coffeescript
when 'asset' if message.asset.original?.image? then 'photo' else 'file'
```

Comment (gregor) [reply app/script/conversation/ConversationRepository.coffee]: No, otherwise message with asset metadata will be counted as files.

Comment (lipis) [reply app/script/conversation/ConversationRepository.coffee]: Then just fix the indentation.. :)

Comment (gregor) [reply app/script/conversation/ConversationRepository.coffee]: Done :)

**Pull request merged**